src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.4
 ---
 
+ * Deprecated calling `FormErrorIterator::children()` if the current element is not iterable.
```

Comment:
Deprecate [...]

Reply:
Seems like other entries in this file use "deprecated" instead of "deprecate".
Should this be changed anyway?

Reply:
We've changed our ChangeLog style with 5.3. Sorry for the confusion, but @xabbuh is right.

Reply:
Thank you for clarification.